src/components/SlateEditor/plugins/embed/SlateVideo.jsx

```diff
@@ -33,53 +33,64 @@ class SlateVideo extends React.PureComponent {
       attributes,
       figureClass,
       onRemoveClick,
+      t,
       ...rest
     } = this.props;
     const src = `//players.brightcove.net/${config.brightCoveAccountId}/${
       config.brightcovePlayerId
     }_default/index.min.js`;
     return (
-      <Figure id={embed.videoid} {...attributes}>
-        <DeleteButton stripped onClick={onRemoveClick} />
+      <div className="c-figure" {...attributes}>
+        <FigureButtons
+          tooltip={t('form.video.remove')}
+          onRemoveClick={onRemoveClick}
+          embed={embed}
+          t={t}
```

Comment:
Vits å sende ned t? Og ikke bruker ndla-i18n?

Reply:
Fordi jest feilet for nestede komponenter som tok inn ndla-i18n